ceph.spec.in

```diff
@@ -47,7 +47,12 @@ Release:	@RPM_RELEASE@%{?dist}
 Epoch:		1
 Summary:	User space components of the Ceph file system
 License:	GPL-2.0
+%if 0%{?fedora} || 0%{?rhel}
```

Comment:
We can simply drop the "Group:" tag entirely on RHEL and Fedora. I just checked, and RHEL 5 is the only distro that requires it. (and we don't support RHEL 5)